static/app/views/insights/browser/webVitals/components/browserTypeSelector.tsx

```diff
@@ -16,6 +16,7 @@ import {SpanIndexedField} from 'sentry/views/insights/types';
 const LabelContainer = styled('div')`
   display: flex;
   gap: ${space(1)};
+  min-width: 130px;
```

Comment:
Instead of pinning a width here, we could also use `width: max-content` so additional options don't end up overflowing and have space.